docs/cog_guides/warnings.rst

```diff
@@ -0,0 +1,403 @@
+.. _warnings:
+
+========
+Warnings
+========
+
+This is the cog guide for the warnings cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load warnings
+
+.. _warnings-usage:
+
+-----
+Usage
+-----
+
+Warn misbehaving users and take automated actions.
+
+
+.. _warnings-commands:
+
+--------
+Commands
+--------
+
+.. _warnings-command-actionlist:
+
+^^^^^^^^^^
+actionlist
+^^^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]actionlist 
+
+**Description**
+
+List all configured automated actions for Warnings.
+
+.. _warnings-command-mywarnings:
+
+^^^^^^^^^^
+mywarnings
+^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]mywarnings 
+
+**Description**
+
+List warnings for yourself.
+
+.. _warnings-command-reasonlist:
+
+^^^^^^^^^^
+reasonlist
+^^^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]reasonlist 
+
+**Description**
+
+List all configured reasons for Warnings.
+
+.. _warnings-command-unwarn:
+
+^^^^^^
+unwarn
+^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]unwarn <member> <warn_id> [reason]
+
+**Description**
+
+Remove a warning from a member.
+
+**Arguments**
+
+* ``<member>``: The member to warn. |member-input-quotes|
+* ``<warn_id>``: The warning ID to remove from the member.
+* ``[reason]``: The reason for unwarning this member.
+
+.. _warnings-command-warn:
+
+^^^^
+warn
+^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]warn <member> [points=1] <reason>
+
+**Description**
+
+Warn the user for the specified reason.
+
+**Arguments**
+
+* ``<member>``: The member to warn. |member-input-quotes|
+* ``<points>``: The number of points the warning should be for. If no number is supplied, 1 point will be given. Pre-set warnings disregard this.
+* ``<reason>``: The reason for the warning. This can be a registered reason if it exists or a custom one is created by default.
```

Comment:
This second sentence is a bit awkward, perhaps break it into 2 sentences? You also should specify that custom reasons are only allowed if `[p]warningset allowcustomreasons` is set.